packages/plugins/i18n/server/services/localizations.js

```diff
@@ -11,7 +11,14 @@ const { getService } = require('../utils');
 const assignDefaultLocale = async (data) => {
   const { getDefaultLocale } = getService('locales');
 
-  if (isNil(data.locale)) {
+  if (isArray(data) && data.some((entry) => !entry.locale)) {
+    const defaultLocale = await getDefaultLocale();
+    data.forEach((entry) => {
+      if (isNil(entry.locale)) {
+        entry.locale = defaultLocale;
+      }
```

Comment:
I have a question aboutthe isNil, is there a difference between this and
`entry.locale ??= defaultLocale` or `entry.locale = entry.locale || defaultLocale`

Not that I have a preference, I am curious to know if isNil does more things that I do not know.

Reply:
Good idea, I'm not used to the new syntax in node. I'll use `??=`! :)

Reply:
Oh I cannot, it was introduced in NodeJS 15. I'll use `entry.locale = entry.locale || defaultLocale` then

Reply:
ooh, didn't know about that, such as shame! It's a nice operator